pkg/cloudprovider/aws/instance.go

```diff
@@ -128,11 +128,19 @@ func (p *InstanceProvider) launchInstances(ctx context.Context, constraints *v1a
 			},
 		},
 	}
+
+	fleetContext := injection.GetOptions(ctx).AWSEC2FleetContext
```

Comment:
I think you can just add `Context` to the CreateFleetInput struct above. The default should be nil anyways, so doing this should be equivalent

```
...
Context: aws.String(injection.GetOptions(ctx).AWSEC2FleetContext)
```